lib/arraySum.js

```diff
@@ -1,7 +1,11 @@
-// Write a function that sums all the numerical elements of an array
-
 function arraySum(array) {
-  // Write code here
+  var sum = 0;
+  Array.prototype.forEach.call(array, (elem) => {
+    if (typeof elem == "number" && !Number.isNaN(elem) && Number.isFinite(elem)) {
+      sum += elem;
+    }
+  });
+  return sum;
```

Comment:
I'm not sure what the maintainers want but have you tried doing this with `array.reduce` you may find it to be a bit simpler.

Reply:
@ktilcu thanks for the suggestion. I have used the Array.reduce for brevity.